src/coverlet.console/Program.cs

```diff
@@ -15,6 +15,42 @@ namespace Coverlet.Console
 {
     class Program
     {
+        /// <summary>
+        /// Exit Codes returned from Coverlet console process.
+        /// </summary>
+        enum CommandExitCodes
+        {
+            /// <summary>
+            /// Indicates successful run of dotnet test without any test failure and coverage percentage above threshold if provided.
+            /// </summary>
+            Success = 0,
+
+            /// <summary>
+            /// Indicates test failure by dotnet test.
+            /// </summary>
+            TestFailed = 1,
+
+            /// <summary>
+            /// Indicates coverage percentage is below given threshold for one or more threshold type.
+            /// </summary>
+            CoverageBelowThreshold = 2,
+
+            /// <summary>
+            /// Indicates test failure by dotnet test and coverage percentage is below given threshold for one or more threshold type.
+            /// </summary>
+            TestFailedAndCoverageBelowThreshold = TestFailed + CoverageBelowThreshold,
+
+            /// <summary>
+            /// Indicates exception occurred during Coverlet process.
+            /// </summary>
+            Exception = 101,
+
+            /// <summary>
+            /// Indicates missing options or empty arguments for Coverlet process.
+            /// </summary>
+            CommandParsingException = 102
+        }
+
```

Comment:
I created for enum for exit codes to have code level documentation for reference and for ease of change in future.